test/unit/spec/util/BootstrapDialogSpec.js

```diff
@@ -1,7 +1,7 @@
 define(['jquery', 'util/bootstrapDialog'], ($, Dialog) => {
     'use strict';
     const $simpleBody = $('<div>').append('This is a body text'),
-        simpleTitle = 'Title',
+        simpleTitle = 'Simple Modal Title',
```

Comment:
Changed the title to make the source of the modal easier to find in the test specs.